app/models/manageiq/providers/hawkular/inventory/collector/middleware_manager.rb

```diff
@@ -0,0 +1,86 @@
+module ManageIQ::Providers
+  class Hawkular::Inventory::Collector::MiddlewareManager < ManagerRefresh::Inventory::Collector
+    require 'hawkular/hawkular_client'
```

Comment:
Most of your comments are on code that I copied and pasted to this file. I'm honoring all your suggestions because are great suggestions. :+1: An also because your suggestions make the code much cleaner and, since I'm already doing a refactor, better to clean it up.